src/Mod/PartDesign/Gui/TaskPadPocketParameters.ui

```diff
@@ -16,10 +16,34 @@
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
-     <item row="0" column="0">
-      <widget class="QLabel" name="textLabel1">
+     <item row="1" column="0">
```

Comment:
It's not very important but I prefer when the items are sorted correctly in QGridLayouts. Rather than having 1 here, then 2 then 0. But yes this is probably not visible from QtDesigner.

Reply:
Should be fixed now (wasn't able to see this in Qt Designer unfortunately)